lib/osf-components/addon/components/node-publication-doi/component.ts

```diff
@@ -0,0 +1,18 @@
+import { tagName } from '@ember-decorators/component';
+import { computed } from '@ember-decorators/object';
+import Component from '@ember/component';
+
+import { layout } from 'ember-osf-web/decorators/component';
+import { PublicationDoiManager } from 'osf-components/components/editable-field/publication-doi-manager/component';
+import template from './template';
+
+@tagName('')
+@layout(template)
+export default class NodePublicationDoi extends Component {
+    manager!: PublicationDoiManager;
+
+    @computed('manager.publicationDoi')
+    get publicationDoiUrl() {
+        return `https://doi.org/${this.manager.publicationDoi}`;
```

Comment:
Maybe a `formatDoi` function in `utils/doi`? Ideally it would be idempotent, so you don't need to worry about whether the DOI is already in URL form.